### DISCUSSION TITLE
how to create a repository

### DISCUSSION THREAD

**Discussion by lilyoocode** [evilsocket/opensnitch#1059] [Development]
how do i create a repository private

**Comment by samcuxx** [comment]
To create a private repository on GitHub, follow these steps:

Go to GitHub: https://github.com/ and sign in to your account.
Click the + button in the top-right corner of the page and select New repository.
Give your repository a name and description.
Under Visibility, select Private.
Click Create repository.
That's it! You've now created a private repository. You can now add files and start working on your project.

To create a private repository on GitHub using the command line, follow these steps:

Open a terminal window.
Navigate to the directory where you want to create your repository.
Run the following command:
gh repo create <repository-name> --private
For example, to create a private repository called my-private-repo, you would run the following command:

gh repo create my-private-repo --private
This will create a new directory called my-private-repo and initialize it as a Git repository.

To create a private repository on another code hosting platform, such as GitLab or Bitbucket, please consult the documentation for that platform.

Please note that private repositories are only accessible to you and the people you explicitly share access with. This makes them a good choice for storing sensitive data or code that you don't want to share with the public.
